CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ Draft Pull Requests is a feature provided by GitHub as a means to communicate wi
 
 Steps to creating a Draft Pull Request:
 
-1. Commit and push your new changes into the remote repository.
+1. Commit and push your new changes into the remote repository. Please refer to [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional) for your commit message.
```

Comment:
Please let the contributors know that they can use [commitlint.io](https://commitlint.io) to assist them in composing the commit message.

Reply:
Okay. I just pushed for the additional info